docs/content/docs/start/react-tutorial.md

```diff
@@ -0,0 +1,295 @@
+---
+title: 'Tutorial: Create a Fluid Framework application with React'
+menuPosition: 4
+---
+
+In this tutorial, you'll learn about using the Fluid Framework by building a simple application that enables every client of the application to change a dynamic time stamp on itself and all other clients almost instantly. You'll also learn how to connect the Fluid data layer with a view layer made in [React](https://reactjs.org/). The following image shows the time stamp application open in four browsers. Each has a button labelled **click** and beside it a UNIX Epoch time. The same time in all four. The cursor is on the button in one browser.
+
+![Four browsers with the Timestamp app open in them.](images/Four-clients-1.PNG)
+
+The following image shows the same four clients one second after the **click** button was pressed. Note that the timestamp has updated to the very same time in all four browsers.
+
+
+![Four browsers with the Timestamp app open in them one second after the button has been pushed.](images/Four-clients-2.PNG)
```

Comment:
For the images, I'll upload them to a CDN manually and we can use the absolute URLs here. We're trying to avoid committing binaries.

Reply:
> For the images, I'll upload them to a CDN manually and we can use the absolute URLs here. We're trying to avoid committing binaries.

OK. Can I leave the image references "as is" and leave it to you to fix them later?

Reply:
Yeah, that's fine. Where are the images themselves though? I don't see them in the PR.

Reply:
Strange. I added an images folder to FluidFramework\docs\content\docs\start, but my git client doesn't see it. They don't show up when I run git status. I'll email them to you while I try to figure this out.